docs/src/main/asciidoc/security-getting-started-tutorial.adoc

```diff
@@ -54,12 +54,20 @@ You can find the solution in the `security-jpa-quickstart` link:{quickstarts-tre
 
 == Create and verify the Maven project
 
+ifndef::quarkus-security-jpa-reactive[]
 For Quarkus Security to be able to map your security source to Jakarta Persistence entities, ensure that the Maven project in this tutorial includes the `security-jpa` or `security-jpa-reactive` extension.
+endif::quarkus-security-jpa-reactive[]
+ifdef::quarkus-security-jpa-reactive[]
+For Quarkus Security to be able to map your security source to Jakarta Persistence entities, ensure that the Maven project in this tutorial includes the `security-jpa` extension.
+endif::quarkus-security-jpa-reactive[]
 
 [NOTE]
 ====
 xref:hibernate-orm-panache.adoc[Hibernate ORM with Panache] is used to store your user identities, but you can also use xref:hibernate-orm.adoc[Hibernate ORM] with the `security-jpa` extension.
+
+ifndef::quarkus-security-jpa-reactive[]
```

Comment:
Does `ifndef` prohibit referencing the reactive extensions ?

Reply:
This ifndef block displays the content in the community docs and hides it in the product docs. 
I have applied this here because the `quarkus-security-jpa-reactive` extension is not supported in the product.